OneCommunityGlobal/HighestGoodNetworkApp
# Changed from classification to category

Changed “Classification” to “Category” in task edit view.
How to test:
Admin Login → Other Links → Projects → Click WBS Icon by project chosen → Choose WBS → Add or Edit Task
Make sure that the field name is category and not classification
